packages/markdown/remark/src/index.ts

```diff
@@ -79,18 +84,20 @@ export async function renderMarkdown(
 		parser.use([[plugin, pluginOpts]]);
 	});
 
-	if (scopedClassName) {
-		parser.use([scopedStyles(scopedClassName)]);
-	}
+	if (!isPerformanceBenchmark) {
```

Comment:
Can't we use only one single `if (!isPerformanceBenchmark) {}` block?

Reply:
Not if we want to apply plugins in the right order! If we truncate to a single `if`, plugins outside the `if` will apply out of order. We could implement a sorter by name, but seems a bit much